tests/get_or_create_race/models.py

```diff
@@ -0,0 +1,12 @@
+from django.db import models
```

Comment:
We don't need to create a new test folder for this fix and can use `get_or_create` for example
We should try to reuse existing models where we can